.gitignore

```diff
@@ -42,4 +42,11 @@ yarn-error.log*
 bun.lockb
 
 # Tasks
-tasks.md
\ No newline at end of file
+tasks.md
+
+# Log files
+deep-research-mcp.log
```

Comment:
It's better to use wildcard here: `*.log`
This will ensure that any file with `.log` extension will be ignored in future.